### DISCUSSION TITLE
Inquiry Regarding Application for Google Season of Docs Project

### DISCUSSION THREAD

**Discussion by Abhijay007** [r-universe-org/help#395] [Q&A] [answered]
Hi R-universe Community,

My name is Abhijay Jain, and I am reaching out to express my interest in the Google Season of Docs project titled "Documenting R software publication with R‐universe," which I came across while browsing through the available projects on the site.

After thoroughly reading about the project and its objectives, I strongly believe that I can make a meaningful contribution given my relevant experience and skills. However, I encountered difficulty in locating the application link on the project's information page, as there is not much info provided around how to apply for the project [here](https://github.com/r-universe-org/gsod-2024/wiki/GSOD-2024-proposal:-Documenting-R-software-publication-with-R%E2%80%90universe.)

Would it be possible for someone to provide guidance on how to proceed with the application process? Any assistance or direction regarding this matter would be greatly appreciated.

Thank you for your time and consideration. I am eager to contribute to the success of this project and look forward to hearing from you soon.

Warm regards,
Abhijay Jain 

cc : @noamross , @maelle

**Comment by noamross** [comment, marked as answer]
Hello @Abhijay007. Thank you for your interest. However, we have already selected a technical writer for this project.

**Reply by Abhijay007** [reply]
Okay, got it, thanks for your response @noamross

**Comment by johnwinifred** [comment]
Hi My name is Winifred John,  I understand that you have a technical writer working on the project but do you still accept volunteers?. I would love to still contribute even as a volunteer.
Thank you

**Reply by noamross** [reply]
Thank you for inquiring @johnwinifred! Once this project kicks off in a month or so we'll have an open repository and we will have some specific calls for participation.  In the meantime, you may want to look at our Contributing Guide: https://contributing.ropensci.org/ , which outlines a number of ways to contribute to rOpenSci's many projects, including for documentation.

**Reply by johnwinifred** [reply]
Okay @noamross . Thank you. I will look through the guide and in the meantime see how I can contribute to the project.
